modules/vpn_aws_gcp/aws.tf

```diff
@@ -0,0 +1,44 @@
+resource "aws_vpn_gateway" "a" {
```

Comment:
Can we make this VPN Gateway be conditionally created? In case the user already has an existing VPN gateway for their VPC and they want to "bring their own".

Reply:
True, i will do that

Reply:
I think it is not worth the effort as you would have to check if the gateway is available for use:
1. Not attached to any VPC
2. Not used by any VPN connections

Reply:
`use_existing_aws_vgw = true` and the module will locate the vgw attached to the aws vpc. Please see if this  looks good to you.